templates/index.html

```diff
@@ -63,7 +63,7 @@
 <body>
   <button id="reload-btn" class="reload-btn fixing-top">reload page</button>
   <div class="centering">
-    <form mathod="post" action="sub" name="post-form" class="fixing-bottom">
+    <form method="post" action="sub" name="post-form" class="fixing-bottom">
```

Comment:
feel free to cherry-pick this commit.

Reply:
the other commits are (functionally) trivial.
this one needs to be merged.

Reply:
@RandyMcMillan very very thanks to your contribution!